### ISSUE TITLE
Drow Ranger: Arcana + Reapers' Wreath effect

### ISSUE THREAD

**Issue by gillesdegottex** [jeffhill/Dota2#931]
### Cosmetic Item Name

Drow Ranger: Arcana + Reapers' Wreath effect

### Description

* Select Demo Terrain (might not be necessary)
* Select Drow Arcana
* Select Reapers' Wreath effect for Marksmanship's ability

Result: Please see screenshots below.

### Example Match ID

999

### Screenshots

Why not actually
![Screenshot from 2022-06-09 13-26-44](https://user-images.githubusercontent.com/4386879/172836638-c5344baf-3013-4fbe-aab0-8cd2385b47a3.png)

Surely not intended
![Screenshot from 2022-06-09 13-26-57](https://user-images.githubusercontent.com/4386879/172836645-bc5ac892-3c66-48b9-a544-32715b9fd679.png)

**Comment by yellow-core** [comment]
They fixed the overlaping bow, but didn't implement Reapers' Wreath Marksmanship attack with Arcana. It still has the Arcana effect. Shouldn't the attack effect of the immortal be present (instead of Dread Retribution's) when Marksmanship procs?

**Comment by gillesdegottex** [comment]
Yes, I guess the arrow effect is expected, not the stacking of the bows (looks like a confusion on a tag)

**Comment by syn750** [comment]
actually  liked the stacking  effect like in the first  screenshot but  clearly  a bug, please fix the immortal style not working  on the arcana

**Comment by gillesdegottex** [comment]
Yeah, the "Why not actually" wasn't even a joke. It's not bad at all.
The two things that are objectively bugy are:
* The missalignment of the bow and the crossbow.
* The missing arrow effect.

**Comment by tsukismet** [comment]
made a github account specifically for this issue. thankful that it is partly solved, however, the effect still prioritizes the arcana when the when the bow effect is equipped. hopefully they solve it soon
